include/libcyphal/executor.hpp

```diff
@@ -102,7 +102,7 @@ class IExecutor
         ///
         /// Size is chosen arbitrary, but it should be enough to store any callback implementation.
         ///
-        static constexpr std::size_t MaxSize = (sizeof(void*) * 12) + sizeof(Function);
+        static constexpr std::size_t MaxSize = (sizeof(void*) * 16) + sizeof(Function);
```

Comment:
Our `AwaitableNode`-s (which are inherited from `CallbackNode`) now hold couple extra fields (like `fd`, polling flags, etc), so unfortunately we need even bigger footprint to fit everything inside of `Callback::Any` unbounded variant - this is tradeoff for stack allocated callbacks.